starter-code/src/functions-and-arrays.js

```diff
@@ -11,10 +26,27 @@ var words = [
   'crackpot'
 ];
 
+
+function findLongestWord(wordsArray){
+  var longestWord = wordsArray[0];
+  for(var i = 0; i< wordsArray.length; i++){
+    if(wordsArray[i].length > longestWord.length){
+      longestWord = wordsArray[i];
+    }
+  }
+  return longestWord;
+}
+
+
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray(oneNumber, secondNumber){
+  return oneNumber+secondNumber; 
+ }
```

Comment:
This solution would be correct when you add two numbers, but as you named your function sumArray, we expect an array of numbers. In that case we would change it to something like this:
```
function sumArray(array) {
  var sum = 0;
  for (var x = 0; x < array.length; x++) {
    sum += array[x];
  }
  return sum;
}
```